models/issues/review.go

```diff
@@ -247,7 +247,7 @@ func (r *Review) TooltipContent() string {
 		}
 		return "repo.issues.review.official"
 	case ReviewTypeComment:
-		return "repo.issues.review.comment"
+		return "repo.issues.review.commented"
```

Comment:
I didn't want to change the older `repo.issues.review.comment` as it has been there for a long time and it will affect users with custom templates. Open to other ways of handling this.